skorch/tests/test_classifier.py

```diff
@@ -255,6 +255,11 @@ def test_predict_predict_proba(self, net, data, threshold):
         y_pred_proba = net.predict_proba(X)
         assert y_pred_proba.shape == (X.shape[0], 2)
 
+        # the tests below check that we don't accidentally apply sigmoid twice,
+        # which would result in all probas being withiin [0.2689, 0.7311].
+        assert (y_pred_proba < 0.26).any()
```

Comment:
Is this suppose to be `expit(0) = 0.5` on the lower bound? ([scipy.special.expit](https://docs.scipy.org/doc/scipy/reference/generated/scipy.special.expit.html))

```suggestion
        assert (y_pred_proba < expit(0)).any()
```

Reply:
Indeed, we are comparing to `expit(1)` and `expit(-1)`. The loewr bound is not `expit(0)` because we create the probabilities as `torch.stack((1 - prob, prob), 1)`, so the lowest value that could be achieved by applying sigmoid twice is `1 - expit(1)`, which is equal to `expit(-1)`.

Reply:
Can you add this information in a code comment?